pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -224,6 +224,25 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 	} else {
```

Comment:
The comment on line 220 leads me to believe that the initial capacity info gathered may not be completely accurate.